sycl/test-e2e/DeviceArchitecture/device_architecture_unknown_on_host.cpp

```diff
@@ -0,0 +1,28 @@
+// REQUIRES: accelerator
+
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
+
+// Test checks that device_architecture extension implementation correctly
+// handles unsupported HW. The unsupported HW in this test is any FPGA device,
+// as FPGA currently is not supported at all by the device_architecture
+// extension.
+
+#include <sycl/detail/core.hpp>
+
+int main() {
+  sycl::queue q;
+  sycl::device dev = q.get_device();
+
+  sycl::ext::oneapi::experimental::architecture arch = dev.get_info<
+      sycl::ext::oneapi::experimental::info::device::architecture>();
+
+  assert(arch == sycl::ext::oneapi::experimental::architecture::unknown);
+  // device::ext_oneapi_architecture_is(syclex::architecture::unknown) should
+  // return true if the device does not have a known architecture
+  assert(dev.ext_oneapi_architecture_is(arch));
+
+  // no exceptions are expected in this test
```

Comment:
```suggestion
  // No exceptions are expected in this test.
```

Reply:
@maarquitos14 I expect these two suggested changes should be nits and do not block approving any PR because a) the suggested changes are not functional, b) there are no any typos or poorly written sentences in these places in code.

Reply:
Yes, these are nits, and I would not block the PR because of this. However, I'd like to point out that these are style changes enforced by LLVM Guidelines: https://www.llvm.org/docs/CodingStandards.html#commenting

Is there any reason why these can't be applied? If it's not to rerun CI, I think because of the reasons you mentioned we don't need to rerun CI, as it already passed. So, IMO, we can simply apply the changes and merge.